components/escaped-identifier-mapper/src/main/java/com/tibco/ep/sb/contrib/escapedidentifiermapper/EscapedFieldNameCleaner.java

```diff
@@ -0,0 +1,148 @@
+package com.tibco.ep.sb.contrib.escapedidentifiermapper;
```

Comment:
Missing the copyright:
`/*
* Copyright © 2020. TIBCO Software Inc.
* This file is subject to the license terms contained
* in the license file that is distributed with this file.
*/`